Sources/XMTP/Crypto.swift

```diff
@@ -35,8 +35,14 @@ enum Crypto {
 		}
 
 		var ciphertext = CipherText()
-
-		ciphertext.aes256GcmHkdfSha256.payload = payload.ciphertext + payload.tag
```

Comment:
This code left us with `ciphertext.aes256GcmHkdfSha256.payload` being composed of Data subscripts: https://forums.swift.org/t/data-subscript/57195

So subsequent operations like `ciphertext.aes256GcmHkdfSha256.payload[0]` would crash with EXC_BAD_INSTRUCTION